pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -28,9 +28,11 @@ import (
 type State string
 
 const (
-	GA           = "General Availability" // By default, GAed feature gates are considered enabled and no-op.
-	Deprecated   = "Deprecated"           // The feature is going to be discontinued next release
-	Discontinued = "Discontinued"
+	GA             = "General Availability" // By default, GAed feature gates are considered enabled and no-op.
+	Deprecated     = "Deprecated"           // The feature is going to be discontinued next release
+	Discontinued   = "Discontinued"
+	WarningPattern = "feature gate %s is deprecated (feature state is %s), therefore it can be safely removed and is redundant. " +
```

Comment:
nit: I would use %q (wrap with quotes) it reads better IMO

Not a must

Reply:
Done